src/controllers/observation-controller.ts

```diff
@@ -104,10 +102,42 @@ export default class ObservationController extends AbstractController {
     }
   };
 
-  private getAggregations = async (_req: Request, res: Response, next: NextFunction) => {
+  private getAggregations = async (req: Request, res: Response, next: NextFunction) => {
     try {
-      const aggregations = await getAggregations(this.observations);
-      res.json({ ...aggregations });
+      const { lang = 'eng' }: { lang: string } = req.query;
+      const langOrigin = LocaleOrigin[lang] ? LocaleOrigin[lang] : 'desc_eng';
+
+      const paramsAggregation = parseWhereParams(req.user, req.query);
+      const observations = await this.observations.find({ ...paramsAggregation });
+      const aggregatinMap: AggregationsMap = {};
```

Comment:
Redundant annotation, please merge with next line

Reply:
Don't pay attention that `outdated`, it's because my commit. Still actual

Reply:
couldn't remove it due to ts ling error.

Reply:
Please take a look: 49c42d6 -- no any error
@SerafimovichEugene